src/main/java/net/dv8tion/jda/core/hooks/ListenerAdapter.java

```diff
@@ -751,6 +749,11 @@ else if (event instanceof GenericCallVoiceEvent)
                 onGenericCallVoice((GenericCallVoiceEvent) event);
 
             //Client Only Generic Events
+            //Subclass of GenericGroupEvent must be in different if/else block
+            // cannot be in block above due to GenericGroupMessageReactionEvent being a child
+            if (event instanceof GenericGroupMessageEvent)
+                onGenericGroupMessage((GenericGroupMessageEvent) event);
+
             if (event instanceof GenericRelationshipEvent)
```

Comment:
should this be an `else if` ?

Reply:
GenericGroupMessageEvent is a subclass of GenericGroupEvent which is handled in the following if/else block (see comment above the if)